tests/manager/manager-ctor-ssl-deprecated-002.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): Test deprecated options
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_ssl(['OpenSSL', 'LibreSSL']); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+echo raises(
+    function () {
+        new MongoDB\Driver\Manager('mongodb://127.0.0.1/', [], ['capath' => 'foo']);
```

Comment:
Had to extract the test for `capath` to a separate test file as it needs to be skipped when compiled against Secure Channel or Secure Transport.

Reply:
This should technically be:

```php
['context' => stream_context_create(['ssl' => ['capath' => 'foo']])]
```

Reply:
I've added a second test for this. This way, we can separately test the behaviour of specifying the `capath` driver option and nesting it within a stream context. The latter causes a single deprecation message about the "context" parameter to be printed, after which the tests stop checking deprecation messages. Checking multiple messages (as is the case here) does not work.

Reply:
`capath` was never a [documented](https://www.php.net/manual/en/mongodb-driver-manager.construct.php) driver option on its own, so this is mainly testing the internal behavior where context options are merged up. That said, I don't mind having two tests, so long as there's one for the documented behavior we might expect some user(s) to still employ.

I've also acknowledged that `raises()` stops after the first notice, so we can't test for two deprecation messages at once.